source/transfer_list.rst

```diff
@@ -951,6 +951,48 @@ software running in Secure, Non-Secure, or Realm modes.
      - hdr_size
      - Holds a single `entry_point_info` structure.
 
+**FF-A SP binary (XFERLIST_FFA_SP_BINARY)**
+
+This entry holds a reference to an FF-A secure partition's binary.
+
+Hafnium SPMC supports the SP's artefacts to be packaged as a Transfer List
+generated at build time (with TF-A's tlc tool). The respective Transfer List
+contents: SP binary/image, the FF-A manifest, and other optionl resoures
+(e.g. HOB).
+
+Under the the described context, this TE type is for an SPMC implementation
```

Comment:
fix  'the the'

Reply:
Suggest removing "Under the the described context," (if you accept my other comment).